### PR TITLE
Update create-custom-column-type.md

### PR DESCRIPTION
**Pull request by xiropot**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | 8.x
| Description?  | How to create custom column type for Symfony Grid controller

### CODE REVIEW

File: development/components/grid/tutorials/create-custom-column-type.md [hunk 1]
```diff
@@ -5,8 +5,73 @@ weight: 9
 ---
 
 # How to create a custom Column Type
+It is highly likely that you are struggling to develop a module based on the new standards. You want to display some kind of data in a way not already predicted by the core team. No worries. Let's have a look.
```
Comment (thomasnares) [inline development/components/grid/tutorials/create-custom-column-type.md]:
```suggestion

You may need to display some kind of data in a specific format. This is how to do it: 
```

File: development/components/grid/tutorials/create-custom-column-type.md [hunk 2]
```diff
@@ -5,8 +5,73 @@ weight: 9
...
+```
+->add(
+    (new HtmlColumn('some_column'))
+        ->setName($this->trans('Print voucher', [], 'Modules.Yourcoolmodule.Admin'))
+        ->setOptions([
+            'field' => 'identifier_column'
+        ])
+)
+```
+## Step 3. Define the respective .html.twig
```
Comment (thomasnares) [inline development/components/grid/tutorials/create-custom-column-type.md:77]:
```suggestion

## Step 3. Define the respective .html.twig
```

File: development/components/grid/tutorials/create-custom-column-type.md [hunk 3]
```diff
@@ -5,8 +5,73 @@ weight: 9
...
+            ])
+            ->setDefaults([
+                'clickable' => false,
+            ])
+            ->setAllowedTypes('field', 'string')
+            ->setAllowedTypes('clickable', 'bool');
+    }
+}
+```
+## Step 2. Use & call the new type
```
Comment (thomasnares) [inline development/components/grid/tutorials/create-custom-column-type.md:56]:
```suggestion

## Step 2. Use & call the new type

```

File: development/components/grid/tutorials/create-custom-column-type.md [hunk 4]
```diff
@@ -5,8 +5,73 @@ weight: 9
 ---
 
 # How to create a custom Column Type
+It is highly likely that you are struggling to develop a module based on the new standards. You want to display some kind of data in a way not already predicted by the core team. No worries. Let's have a look.
 
-{{% notice tip %}}
-Do you know how to do it? Please feel free to contribute to the docs!
-{{% /notice %}}
+## Step 1. Define the new column type
```
Comment (thomasnares) [inline development/components/grid/tutorials/create-custom-column-type.md:12]:
```suggestion

## Step 1. Define the new column type

```

File: development/components/grid/tutorials/create-custom-column-type.md [hunk 5]
```diff
@@ -5,8 +5,73 @@ weight: 9
...
+Let's assume that we need to make an html column to fire some js script.
+Create the following file:
```
Comment (thomasnares) [inline development/components/grid/tutorials/create-custom-column-type.md:15]:
```suggestion
Create the following file:

```

File: development/components/grid/tutorials/create-custom-column-type.md [hunk 6]
```diff
@@ -5,8 +5,73 @@ weight: 9
 ---
 
 # How to create a custom Column Type
+It is highly likely that you are struggling to develop a module based on the new standards. You want to display some kind of data in a way not already predicted by the core team. No worries. Let's have a look.
 
-{{% notice tip %}}
-Do you know how to do it? Please feel free to contribute to the docs!
-{{% /notice %}}
+## Step 1. Define the new column type
+Let's assume that we need to make an html column to fire some js script.
```
Comment (thomasnares) [inline development/components/grid/tutorials/create-custom-column-type.md]:
```suggestion
Let's assume that we need to make an html column to display an object's ID within a `&lta>` html tag: 
```

**Review by thomasnares — CHANGES_REQUESTED** [review]
Thanks @tivuno, i made a few suggestions 👍

**Review by thomasnares — APPROVED** [review]

**Comment by xiropot** [comment]
@thomasnares Do I need to do anything else or its up to you now?

**Comment by thomasnares** [comment]
looks good to me, lets wait another maintainer approval to merge it 👍 
thanks again

**Review by matks — APPROVED** [review]

**Comment by matks** [comment]
Thank you @tivuno

**Comment by matks** [comment]
@tivuno Would you consider submitting another PR to update `1.7.x` branch?

**Comment by xiropot** [comment]
@matks i will do as soon as i find time. Even though i do believe its 99.99% the very same procedure.

**Pull request merged**